src/eei.cpp

```diff
@@ -476,7 +476,7 @@ namespace HeraVM {
       if (call_result.release)
         call_result.release(&call_result);
 
-      return Literal((call_result.status_code == EVM_SUCCESS) ? 1 : 0);
+      return Literal((call_result.status_code == EVM_SUCCESS) ? (uint32_t)1 : (uint32_t)0);
```

Comment:
Is this cast needed? Compilers usually handle literals nicely.
If needed, write it as `uint32_t(1) : uint32_t(0)`.

Reply:
Needed because it has a lot of overloads and this is to ensure it is always treated as 32 bit.